Public/Start-ADReporting.ps1

```diff
@@ -1,5 +1,6 @@
 function Start-ADReporting () {
     param (
+        [hashtable]$LoggerParameters,
```

Comment:
Check if LoggerParamters exists? If it doesn't stop? Maybe we should simply require this?